server/src/main/java/org/opensearch/common/settings/FeatureFlagSettings.java

```diff
@@ -38,6 +38,7 @@ protected FeatureFlagSettings(
         FeatureFlags.APPLICATION_BASED_CONFIGURATION_TEMPLATES_SETTING,
         FeatureFlags.TERM_VERSION_PRECOMMIT_ENABLE_SETTING,
         FeatureFlags.ARROW_STREAMS_SETTING,
-        FeatureFlags.MERGED_SEGMENT_WARMER_EXPERIMENTAL_SETTING
+        FeatureFlags.MERGED_SEGMENT_WARMER_EXPERIMENTAL_SETTING,
+        FeatureFlags.TERMS_AGGREGATION_OPTIMIZATION_ENABLE_SETTING
```

Comment:
This is not a feature hence should not be a feature flag, please change it to either index or cluster setting.